webapp/src/marked.tsx

```diff
@@ -287,6 +287,16 @@ export class MarkedContent extends data.Component<MarkedContentProps, MarkedCont
 
 
     private renderVideo(content: HTMLElement){
+
+        pxt.Util.toArray(content.querySelectorAll('iframe.yt-embed'))
+        .forEach((inlineVideo: HTMLElement)=>{
+
+            let lang=pxt.appTarget.appTheme?.defaultLocale ?? "en";
```

Comment:
```suggestion
            .forEach((inlineVideo: HTMLElement) => {
                let lang = pxt.appTarget.appTheme?.defaultLocale ?? "en";
```

Make sure to check that the style of your code roughly matches existing code -- when spacing is inconsistent or out of place it can be harder to identify errors (e.g. the indentation of this foreach is wrong so it looks like the scope is wrong).

It may seem super small but when spacing is nice and consistent it can be much easier to parse at a glance, especially when you're like through 100s or 1000s of lines of code

Reply:
Good point, should have run 'beautify' to make sure I had the right spacing/indentation. Thanks!